### PR TITLE
Feat/create centralized paths

### PR DESCRIPTION
**Pull request by Albertlarsen**
## Description
Implemented a centralized solution for path management in the react-app so if a path is changed one place it is changed everywhere.

## Related Issue(s)
- #103 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/routes/Router/Router.tsx [hunk 1]
```diff
@@ -1,43 +1,56 @@
...
 import { OverviewPage as GivenOverviewPage } from '@/components/apiDelegation/given/OverviewPage';
 import { OverviewPage as ReceivedOverviewPage } from '@/components/apiDelegation/received/OverviewPage';
 import { NewOrgDelegationPage } from '@/components/NewOrgDelegationPage';
 import { ApiDelegationConfirmationPage } from '@/components/ApiDelegationConfirmationPage';
 
+export enum RouterPath {
+  GivenApiDelegations = 'given-api-delegations',
+  GivenApiDelegationOverview = 'overview',
+  NewGivenApiDelegation = 'pick-api',
+  NewGivenOrgDelegation = 'pick-org',
```
Comment (allinox) [inline src/routes/Router/Router.tsx:15]: Use 'choose' instead of 'pick'?

File: src/routes/Router/Router.tsx [hunk 2]
```diff
@@ -1,43 +1,56 @@
...
-  ],
+  createRoutesFromElements(
+    <Route
+      path='/'
+      errorElement={<ErrorPage />}
+    >
+      <Route
+        path={RouterPath.GivenApiDelegations + '/' + RouterPath.GivenApiDelegationOverview}
+        element={<GivenOverviewPage />}
+      />
```
Comment (allinox) [inline src/routes/Router/Router.tsx:32]: Why does this page not have a `errorElement={<ErrorPage />}` when all the others do?

File: src/rtk/features/overviewOrg/overviewOrgSlice.ts [hunk 3]
```diff
@@ -35,6 +35,7 @@ export interface SliceState {
   loading: boolean;
   overviewOrgs: OverviewOrg[];
   error: string;
+  layout: LayoutState;
```
Comment (allinox) [inline src/rtk/features/overviewOrg/overviewOrgSlice.ts:38]: I thought I removed this from my PR.. 😅

File: src/rtk/features/overviewOrg/overviewOrgSlice.ts [hunk 4]
```diff
@@ -53,6 +54,7 @@ const initialState: SliceState = {
   loading: true,
   overviewOrgs: [],
   error: '',
+  layout: LayoutState.Given,
```
Comment (allinox) [inline src/rtk/features/overviewOrg/overviewOrgSlice.ts:57]: ..and this

File: src/rtk/features/overviewOrg/overviewOrgSlice.ts [hunk 5]
```diff
@@ -172,6 +174,9 @@ const overviewOrgSlice = createSlice({
   name: 'overviewOrg',
   initialState,
   reducers: {
+    setLayout: (state, action) => {
+      state.layout = action.payload;
+    },
```
Comment (allinox) [inline src/rtk/features/overviewOrg/overviewOrgSlice.ts:179]: ..... and this 😅

File: src/components/apiDelegation/received/OverviewPage/OverviewPage.module.css [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+.page{
+  --page-vertical-margins: 8rem;
+  width: 1055px;
+  margin: auto;
+  margin-top: var(--page-vertical-margins);
+  margin-bottom: var(--page-vertical-margins);
+}
+
```
[the remark this answers is not in the document]
Comment (allinox) [reply src/components/apiDelegation/received/OverviewPage/OverviewPage.module.css:17]: Cause I thought I deleted it.. 😅

File: cypress/e2e/api-delegations.cy.tsx [hunk 7]
```diff
@@ -1,6 +1,8 @@
-describe('renders api-delegations', () => {
+import { RouterPath } from '@/routes/Router';
+
+describe(RouterPath.GivenApiDelegations + '/' + RouterPath.ReceivedApiDelegations, () => {
   it('renders correctly', () => {
-    cy.visit('/api-delegations');
+    cy.visit(RouterPath.GivenApiDelegations + '/' + RouterPath.ReceivedApiDelegations);
```
Comment (allinox) [inline cypress/e2e/api-delegations.cy.tsx:5]: I might be misunderstanding something here, but why is the route used here `given-api-delegations/received-api-delegations` ?

File: src/routes/Router/Router.tsx (hunk 2 above)
Comment (Albertlarsen) [reply src/routes/Router/Router.tsx:32]: Because I forgot it :p

File: cypress/e2e/api-delegations.cy.tsx (hunk 7 above)
Comment (Albertlarsen) [reply cypress/e2e/api-delegations.cy.tsx:5]: No you're not. I wobbled with the the urls here. Nice spotted.

**Pull request merged**